src/run-flyway/run-flyway-migrate.ps1

```diff
@@ -18,7 +18,15 @@ $ErrorActionPreference = "Stop";
 
 Write-Information -InformationAction Continue -MessageData "Running migrate..."   
 
-$flywayLocations = "filesystem:`"$(Resolve-Path $pathToMigrationFiles)`""
+$resolvedPaths = New-Object -TypeName "System.Collections.ArrayList"
+$pathToMigrationFiles.Split(",") | ForEach-Object {
+    $resolvedPaths.Add("filesystem:`"$(Resolve-Path $_)`"")
+    Write-Output "resolvedPaths = $resolvedPaths"
+}
+
+$flywayLocations = $resolvedPaths -Join ','
+
+Write-Output "flywayLocations = $flywayLocations"
```

Comment:
This might be confusing to see in the output if people aren't familiar with the internals of the action or flyway itself. What do you think of something like this?

```suggestion
Write-Output "List of migration directories: $flywayLocations"
```

Reply:
I made this change.